pkg/envoy/admin/access/static.go

```diff
@@ -35,39 +39,26 @@ func NewStaticEnvoyAdminAccess(
 
 func mapAccess(users []string, groups []string) accessMaps {
 	m := accessMaps{
-		usernames: map[string]bool{},
-		groups:    map[string]bool{},
+		usernames: make(map[string]struct{}, len(users)),
```

Comment:
probably for the next PR: this looks pretty similar to the other file, should we DRY it up?

Reply:
Yeah wanted to be careful at changing too much (already doing a bit of this...